jbpm-bpmn2/src/test/java/org/jbpm/bpmn2/StartEventTest.java

```diff
@@ -290,17 +287,17 @@ public void beforeProcessStarted(ProcessStartedEvent event) {
         });
         ksession.signalEvent("MySignal", "NewValue");
 
-        assertEquals(1, getNumberOfProcessInstances("Minimal"));
+        Assertions.assertThat(getNumberOfProcessInstances("Minimal")).isEqualTo(1);
         // now remove the process from kbase to make sure runtime based listeners are removed from signal manager
         kbase.removeProcess("Minimal");
         
         try {
             ksession.signalEvent("MySignal", "NewValue");
         } catch (IllegalArgumentException e) {
-            assertEquals("Unknown process ID: Minimal", e.getMessage());
+            Assertions.assertThat(e.getMessage()).isEqualTo("Unknown process ID: Minimal");
```

Comment:
wouldn't this be better replaced with http://joel-costigliola.github.io/assertj/assertj-core-features-highlight.html#exception-assertion

that I recently learned thanks to @rsynek and it's really handy :)

Reply:
+1, it would be good to rewrite the try-catch to lambda-based assertion as shown in the example. It's nice not to be limited to Java 7, isn't it? :)